<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerClientView</class>
 <widget class="QWidget" name="PlayerClientView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPlainTextEdit" name="chatRecord_plainTextEdit_2">
       <property name="plainText">
        <string>chat history:
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_pushButton_3">
       <property name="text">
        <string>clear chat message</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="chatInput_plainTextEdit_3"/>
     </item>
     <item>
      <widget class="QPushButton" name="send_pushButton_3">
       <property name="text">
        <string>send chat message</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>clear network control message</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="ip_lineEdit">
         <property name="text">
          <string>127.0.0.1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="port_spinBox">
         <property name="minimum">
          <number>4000</number>
         </property>
         <property name="maximum">
          <number>4010</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="connect_pushButton">
       <property name="text">
        <string>connect to server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="disconnect_pushButton_2">
       <property name="text">
        <string>disconnect to server</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>add on base_tank</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear_pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>chatRecord_plainTextEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
